pkg/gits/operations/pull_request_op_test.go

```diff
@@ -103,6 +103,87 @@ func TestCreatePullRequest(t *testing.T) {
 		results.PullRequestArguments.Title, "The PR title should contain the old and new versions")
 }
 
+func TestCreatePullRequestsWithLabels(t *testing.T) {
+
+	_, _, _, commonOpts, _ := getFakeClientsAndNs(t)
+
+	testOrgName := "testowner"
+	testRepoName := "testrepo"
+
+	commonOpts.SetGit(gits.NewGitFake())
+
+	gitter := gits_test.NewMockGitter()
+
+	fakeRepo, _ := gits.NewFakeRepository(testOrgName, testRepoName, nil, nil)
+	fakeGitProvider := gits.NewFakeProvider(fakeRepo)
+	fakeGitProvider.User.Username = testOrgName
+
+	o := operations.PullRequestOperation{
+		CommonOptions: &commonOpts,
+	}
+
+	testhelpers.ConfigureTestOptionsWithResources(o.CommonOptions,
+		[]runtime.Object{},
+		[]runtime.Object{},
+		gitter,
+		fakeGitProvider,
+		nil,
+		resources_test.NewMockInstaller(),
+	)
+
+	err := testhelpers.CreateTestEnvironmentDir(o.CommonOptions)
+	assert.NoError(t, err)
+
+	pegomock.When(gitter.HasChanges(pegomock.AnyString())).ThenReturn(true, nil)
```

Comment:
I'd probably extract all of the above into some _setUp_ function of sorts so that it is easier to  find and focus on the actual test here.